src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp

```diff
@@ -232,6 +232,8 @@ bool medVtkViewItkDataImageInteractor::SetViewInput(medAbstractData* data, int l
 
 void medVtkViewItkDataImageInteractor::initParameters(medAbstractImageData* data)
 {
+    d->imageData = data; //used among others for 4D
```

Comment:
i wonder if it solves also the dicom import crash. This class did not find imageData :o

Reply:
About 4D Dicom?

Reply:
indeed, the one i sent you this morning ^^

Reply:
From my side it's ok, I mean it's load without crash.
But I don't now this data and maybe it's not render correctly.